cloudify_rest_client/client.py

```diff
@@ -106,8 +108,12 @@ def has_kerberos(self):
         return bool(HTTPKerberosAuth) and is_kerberos_env()
 
     def has_auth_header(self):
-        return CLOUDIFY_TOKEN_AUTHENTICATION_HEADER in self.headers or \
-               CLOUDIFY_AUTHENTICATION_HEADER in self.headers
+        auth_headers = [CLOUDIFY_TOKEN_AUTHENTICATION_HEADER,
+                        CLOUDIFY_AUTHENTICATION_HEADER,
+                        CLOUDIFY_API_AUTH_TOKEN_HEADER,
+                        CLOUDIFY_EXECUTION_TOKEN_HEADER]
+        return any([header in self.headers
```

Comment:
can skip the `[]`, making this just `return any(header in self.headers for header in auth_headers)`, so that the whole list won't need to be precomputed